config.json

```diff
@@ -0,0 +1 @@
+{"enable_ssl":false,"port":8000,"priv_key":null,"ssl_cert":null,"cache_dir":"/tmp/streaming_cache","metadata_dir":"./metadata","quiet_boot":false,"disable_auth":false,"verbose":false}
```

Comment:
Are humans supposed to be able to read/change this?

Reply:
gonna need to migrate from serde_json to some toml serializer for the config file.